core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -245,6 +244,14 @@ protected void doStop() throws Exception {
       context.setBaseResource(staticResources);
     }
 
+    if(config.getBoolean(RestConfig.ENABLE_GZIP_COMPRESSION_CONFIG)) {
+      FilterHolder gzipFilter = new FilterHolder(AsyncGzipFilter.class);
+      // do not check if .gz file already exists for the requested resource
+      gzipFilter.setInitParameter("checkGzExists", "false");
```

Comment:
without this option the filter adds quite a bit of overhead checking for statically compressed resources. This is especially costly if serving static resources from within a jar file. Given that most of the requests are for dynamic content, it's better to turn if off